content/go/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: 'Conditionals'
+Description: 'Conditionals are used to evaluate if an expression is `true` or `false` and execute code based on that value.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comparison'
+  - 'Booleans'
+  - 'If'
+  - 'Else'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+Conditionals are used to evaluate if an expression is `true` or `false` and execute code based on that value. There are several statements that use conditional expressions to determine what code to execute. The primary one being the `if...else if...else` statements.
+
+Other statements that can use conditional statements are [`switch`](https://www.codecademy.com/resources/docs/go/switch) and [`for`](https://www.codecademy.com/resources/docs/go/loops).
+
+## The `if...else if...else` Statements
```

Comment:
@SSwiniarski  let me know what you think, but I'm wondering if this would be better split up into individual sections for "if", "else" and "else if" like how it's done in the other conditional entries like for [Python](https://www.codecademy.com/resources/docs/python/conditionals) or [C++](https://www.codecademy.com/resources/docs/cpp/conditionals). I noticed it's split up in the [Go learn path](https://www.codecademy.com/learn/learn-go/modules/learn-go-conditionals) too. Then, we can have the pseudo-code/example for each corresponding section.

Reply:
Sounds good.